packages/block-library/src/image/image.js

```diff
@@ -839,6 +843,30 @@ export default function Image( {
 						</ToolsPanelItem>
 					) }
 					{ dimensionsControl }
+					{ ( aspectRatio || scale ) && url && (
```

Comment:
Should we care about `scale`? It seems it's only applied if we have `aspectRatio`.